vic/drivers/image/src/vic_force.c

```diff
@@ -349,6 +349,7 @@ vic_force(void)
             // precipitation in mm/period
             atmos[i].prec[j] *= global_param.snow_dt;
             // pressure in Pa
+            // atmos[i].pressure[j] = atmos[i].pressure[j] *1000;
```

Comment:
I will remove this line, sorry for forgetting.
However the discharges are still way too low, and I suspect this has still something to do with units...